truss/base/trt_llm_config.py

```diff
@@ -17,6 +16,11 @@
     validator,
 )
 
+if TYPE_CHECKING:
+    from truss.base.truss_config import TrussConfig
+
+from truss.base import constants
```

Comment:
sounds dumb, but can we not import constants here and move it to where it's needed. We are happy to import it multiple times.

In briton, we dont have truss installed in this part afaik cc @bdubayah